assignments/treesBFS/levelOrderSuccessor/levelOrderSuccessor.js

```diff
@@ -49,4 +49,3 @@ if (result != null)
 result = find_successor(root, 9)
 if (result != null)
   console.log(result.val)
-
```

Comment:
Not sure why this is still lingering around

Reply:
The issue is that the file being edited is in this path `assignments/treesBFS/levelOrderSuccessor/levelOrderSuccessor.js`. The levelOrderSuccessor file should be added/edited inside your personal folder `henryPuma/assignments...`.

Reply:
Excuse my previous comment, if possible can you try and add that line back in? Let me know if it does not work.